Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/EndRide.hs

```diff
@@ -124,7 +124,7 @@ driverEndRide ::
   ServiceHandle m ->
   Id DRide.Ride ->
   DriverEndRideReq ->
-  m APISuccess.APISuccess
+  m ()
 driverEndRide handle rideId req =
```

Comment:
why are we removing APISuccess?

Reply:
> why are we removing APISuccess?

Why not? It's not up to domain logic to return this anyway